_episodes/01-welcome.md

```diff
@@ -8,8 +8,10 @@ questions:
 - "What will we cover (or not) in this workshop?"
 - "What prior knowledge do workshop participants have?"
 objectives:
+- "Explain what made their favorite class(es) so great." 
```

Comment:
That's not an objective, it's a method.  Maybe the objective should just be "identify a component of successful teaching."